replica/backup.go

```diff
@@ -280,12 +280,6 @@ func preload(d *diffDisk) error {
 		if i == 0 {
 			continue
 		}
-		if i == 1 {
-			// Reinitialize to zero so that we can detect holes in the base snapshot
-			for j := 0; j < len(d.location); j++ {
```

Comment:
Why this has been removed?

Reply:
This is not required as the map is always empty whenever preload is called.

Reply:
lets add comments accordingly @payes with the description, assumptions of inputs and the output of this function